src/components/SimpleChart/SimpleChart.tsx

```diff
@@ -0,0 +1,263 @@
+          case 'Line':
+            return chart.addLineSeries({
+              color: lineColor,
+            })
+          case 'Histogram':
+            return chart.addHistogramSeries({
+              color: accentColorValue,
+              priceFormat: {
+                type: 'volume',
+              },
+            })
+          case 'Bar':
+            return chart.addBarSeries({
+              upColor: topColor,
+              downColor,
+            })
+          case 'Area':
+            return chart.addAreaSeries({
+              lineColor: accentColorValue,
+              lineType: data.length < 20 ? LineType.WithSteps : LineType.Curved,
+              lineWidth: 2,
+              crosshairMarkerRadius: 5,
+              crosshairMarkerBorderColor: opacify(30, brandColor),
+              crosshairMarkerBorderWidth: 3,
+              topColor: opacify(12, accentColorValue),
+              bottomColor: opacify(12, accentColorValue),
+            })
+          case 'Candlestick':
+            return chart.addCandlestickSeries({
+              upColor: topColor,
+              borderUpColor: topColor,
+              wickUpColor: topColor,
+              downColor,
+              borderDownColor: bottomColor,
+              wickDownColor: bottomColor,
+            })
+          case 'Baseline':
+            return chart.addBaselineSeries({
+              topLineColor: topColor,
+              bottomLineColor: bottomColor,
+            })
+          default:
+            return chart.addLineSeries({
+              color: lineColor,
+            })
+        }
+      })()
+
+      newSeries.setData(data)
+      chart.timeScale().fitContent()
+
+      const handleResize = () => {
+        chart.applyOptions({
+          width: chartContainerRef.current?.offsetWidth,
+          height: chartContainerRef.current?.offsetHeight,
+        })
+      }
+
+      const handleCrosshairMove = (event: MouseEventParams) => {
+        if (event.time) {
+          const data = event.seriesData.get(newSeries) as HistogramData<Time>
+          setCrosshairData(data)
+        } else {
+          setCrosshairData(undefined)
+        }
+      }
+
+      chart.subscribeCrosshairMove(handleCrosshairMove)
+
+      window.addEventListener('resize', handleResize)
+
+      return () => {
+        window.removeEventListener('resize', handleResize)
+        chart.remove()
+      }
+    }
+  }, [
+    accentColorValue,
+    boldBorder,
+    brandColor,
+    data,
+    lineColor,
+    seriesType,
+    surfaceColor,
+    textColor,
+  ])
+
+  // Prevent scrolling while scrubbing the chart
+  const handleTouchMove = useCallback(
+    (e: React.TouchEvent<HTMLDivElement>) => e.preventDefault(),
+    [],
+  )
+
+  const chartHeader = useMemo(() => {
+    return (
+      <ChartHeader
+        value={crosshairData?.value ?? lastPrice.value}
+        time={crosshairData?.time as UTCTimestamp}
+        timePlaceholder={formatHistoryDuration(interval)}
```

Comment:
We need to `translate()` this

<img width="136" alt="image" src="https://github.com/shapeshift/web/assets/17035424/c5010233-3199-431e-af5f-22dfa9ff7709">